index.js

```diff
@@ -1,6 +1,43 @@
+const yaml = require('js-yaml')
 const mergeArrayByName = require('./lib/mergeArrayByName')
 
 module.exports = (robot, _, Settings = require('./lib/settings')) => {
+  robot.on('installation', async context => {
```

Comment:
I believe you also need to include the installations repositories [event](https://developer.github.com/v3/activity/events/types/#installationrepositoriesevent) `installation_repositories.added` which is triggered when a repo is added to the installation. This is a different event from installing a GitHub app which is the `installation.created` event.

Reply:
Sorry, I don't think I made this clear: `installation_repositories.added` is still needed as an event to listen for (This event is different from `installation` which is what you currently have). It is triggered when a repo is added to an installation. For example, when you install a bot to your org (that creates an installation and fires `installation` event), and when you add repos to the bot's access later, this fires the `installation_repositories.added` event. 

Also the field name is different to get the repos from the event payload, so watch out for that. https://developer.github.com/v3/activity/events/types/#installationrepositoriesevent

Reply:
@kevo1ution I understood your comment but I decided, for some reason, to seperate the two changes.